pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -284,6 +288,19 @@ type BatchSchedulerConfiguration struct {
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
 }
 
+// Specific SparkUI config parameters
```

Comment:
Can we rename this to `SparkUIConfiguration`, which is easier to understand?

Reply:
changed as you suggested + I changed ExpositionOptions to SparkUIOptions